### PR TITLE
19 token amount input component

### PR DESCRIPTION
**Pull request by ngrujicic**

### CODE REVIEW

File: src/components/atom/Button/index.tsx [hunk 1]
```diff
@@ -43,11 +43,11 @@ const Button = ({ children, onClick, disabled, className, icon, variant, size }:
...
       {icon && showIcon() ? icon : null}
       {children}
-      {isButtonHover && !disabled && variant === "primary-select" ? <DownArrowWhite /> : null}
-      {!isButtonHover && !disabled && variant === "primary-select" ? <DownArrowWhite /> : null}
+      {isButtonHover && !disabled && variant === "primary-select" ? <DownArrowWhite width={16} height={16} /> : null}
+      {!isButtonHover && !disabled && variant === "primary-select" ? <DownArrowWhite width={16} height={16} /> : null}
 
-      {isButtonHover && !disabled && variant === "secondary-select" ? <DownArrowWhite /> : null}
-      {!isButtonHover && !disabled && variant === "secondary-select" ? <DownArrowBlack /> : null}
+      {isButtonHover && !disabled && variant === "secondary-select" ? <DownArrowWhite width={16} height={16} /> : null}
```
Comment (markoMVP85) [inline src/components/atom/Button/index.tsx]: Things like "secondary-select", "primary-select" should be in the enum. Make a global folder if we do not have it and create enum.ts there.

File: src/components/molecule/tokenAmountInput/index.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  className?: string;
+  tokenIcon?: React.ReactNode;
+  isFocused?: boolean;
+  tokenValue?: number;
+  onSetIsFocused: (focus: boolean) => void;
+  onSetTokenValue: (value: number) => void;
+  ref: React.RefObject<HTMLDivElement>;
+};
+
+const TokenAmountInput = React.forwardRef(function TokenAmountInput(
```
Comment (markoMVP85) [inline src/components/molecule/tokenAmountInput/index.tsx]: Why forwardRef here?

File: src/components/molecule/tokenAmountInput/index.tsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    onSetTokenValue,
+    onSetIsFocused,
+    onClick,
+  }: TokenAmountInputProps,
+  ref: React.ForwardedRef<HTMLDivElement>
+) {
+  return (
+    <div
+      ref={ref}
+      className={`relative flex h-[103px] w-[424px] items-center justify-start gap-2 rounded-[8px] bg-[#F3F5FB] px-4 py-6 ${
```
Comment (markoMVP85) [inline src/components/molecule/tokenAmountInput/index.tsx]: we use classname package for all cases that we have conditions in className.

File: src/components/molecule/tokenAmountInput/index.tsx [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+        isFocused ? "border border-solid border-[#E6007A]" : null
+      }`}
+    >
+      <label htmlFor="token-amount" className="absolute top-4 text-[11px] font-normal text-[#797a7e]">
```
Comment (ra79sta) [inline src/components/molecule/tokenAmountInput/index.tsx]: You have font size variables  in tailwind.config

File: src/components/molecule/tokenAmountInput/index.tsx [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+        You pay
+      </label>
+      <NumericFormat
+        id="token-amount"
+        allowNegative={false}
+        fixedDecimalScale
+        displayType={"input"}
+        placeholder={"0"}
+        className="w-full basis-auto bg-transparent font-inter text-[28px] font-bold text-black outline-none"
```
Comment (ra79sta) [inline src/components/molecule/tokenAmountInput/index.tsx]: You have font size variables in tailwind.config

Comment (ra79sta) [inline src/components/molecule/tokenAmountInput/index.tsx]: Font is inter by default

File: src/components/atom/Button/index.tsx (hunk 1 above)
Comment (ngrujicic) [reply src/components/atom/Button/index.tsx]: resio

File: src/components/molecule/tokenAmountInput/index.tsx (hunk 3 above)
Comment (ngrujicic) [reply src/components/molecule/tokenAmountInput/index.tsx]: done

File: src/components/molecule/tokenAmountInput/index.tsx (hunk 2 above)
Comment (ngrujicic) [reply src/components/molecule/tokenAmountInput/index.tsx]: done

**Review by markoMVP85 — APPROVED** [review]

**Pull request merged**